metadata-db-app/src/migrations/058_add_provider_id_to_variables_table.clj

```diff
@@ -0,0 +1,24 @@
+(ns migrations.058-add-provider-id-to-variables-table
+  "Adds `provider_id` column to variables table."
+  (:require
+   [clojure.java.jdbc :as j]
+   [config.mdb-migrate-helper :as h]))
+
+(defn up
+  "Migrates the database up from version 57 to 58."
+  []
+  (println "migrations.058-add-provider-id-to-variables-table up...")
+  ;; At this point in time, UMM-Vars support is still being developed, so we
+  ;; don't care about what's in the database right now, and the new field
+  ;; can't be null, so:
+  (h/sql "TRUNCATE TABLE METADATA_DB.cmr_variables")
+  (h/sql (str "ALTER TABLE METADATA_DB.cmr_variables "
+              "ADD provider_id VARCHAR(255) NOT NULL "
```

Comment:
It looks like we are using 10 characters in some tables and 255 in others for the provider-id. 10 characters is the correct max allowed length:

From `cmr.metadata-db.services.provider-validation`
`(def ^:private ^:const PROVIDER_ID_MAX_LENGTH 10)`